src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -93,26 +96,9 @@ public class RecordService {
   // This class is designed to work with empty metadata config map
   private SnowflakeMetadataConfig metadataConfig = new SnowflakeMetadataConfig();
 
-  /** Send Telemetry Data to Snowflake */
-  private final SnowflakeTelemetryService telemetryService;
-
-  /**
-   * process records output JSON format: { "meta": { "offset": 123, "topic": "topic name",
-   * "partition": 123, "key":"key name" } "content": "record content" }
-   *
-   * <p>create a JsonRecordService instance
-   *
-   * @param telemetryService Telemetry Service Instance. Can be null.
-   */
-  public RecordService(SnowflakeTelemetryService telemetryService) {
-    this.telemetryService = telemetryService;
-  }
-
   /** Record service with null telemetry Service, only use it for testing. */
   @VisibleForTesting
-  public RecordService() {
-    this(null);
-  }
+  public RecordService() {}
```

Comment:
A new constructor accepting `Clock` is going to be introduced.
That's why this constructor is still marked as `@VisibleForTesting`